plugins/disk/quota2percent_

```diff
@@ -0,0 +1,247 @@
+#!/bin/bash
+# -*- sh -*-
+: <<=cut
+
+=head1 NAME
+
+quota2percent - Plugin to show disk usage in percent of quota hard limit.
+
+=head1 APPLICABLE SYSTEMS
+
+All systems with "bash", "quota", "repquota" and "munin"
+Systems with multiple users and individual storage space limitations administered via 'quota'
+
+=head1 CONFIGURATION
+
+The following is the default configuration
+
+  [quota2percent_*]
+  user root
+
+You could define two alert levels and the graph language
+
+  [quota2percent_*]
+  env.warning   [value]        (default: 90)
+  env.critical  [value]        (default: 95)
+  env.language  [en|de|es]     (default: en)
+  env.humanuid  [value]        (default: 1000, only need if there is an other value define for UID_MIN in /etc/login.defs)
+
+=head1 DESCRIPTION
+
+Wild card Plugin for monitoring the utilization of devices with quota rules. 
+A graph is drawn for each user, which shows the usage as a percentage of his hard limit. System accounts (UID <1000) are suppressed. 
+In addition, a graph is displayed which indicates the ratio device size to device coverage. 
+The script repqutoa, usually part of the package quota, is needed. 
+The plugin itself can be stored in any directory. For example, the device sdb1 shell be monitored, a symbolic link must be created
+in the /etc/munin/plugins/ directory as follows: 
+
+=over  
+
+I<<<  ln -s /<path to file>/quota2percent_  quota2percent_sdb1 >>>
+
+=back
+
+=head1 MAGIC MARKERS
+
+  #%# family=auto
+  #%# capabilities=autoconf
+
+=head1 VERSION
+
+17.0210
+
+=head1 HISTORY
+
+V17.0210
+
+  add    env.humanid
+  add    check if device exist
+  add    if no limitations administered via 'quota' for the device the total line is shown only
+  fix    some nitpicking details
+  add    some comments
+
+V17.0131
+
+  add    POD documentation
+  add    env.language
+  add    example graph for Munin Plugin Gallery
+  remove German comments  
+
+V17.0124
+
+  first version, not munin rules conform
+
+=head1 AUTHOR
+
+Jo Hartmann
+
+=head1 LICENSE
+ 
+GPLv2 (L<http://www.gnu.org/licenses/gpl-2.0.html>)
+
+=cut
+
+###################################################
+# Autoconf section                                #
+###################################################
+
+  if [ "$1" = "autoconf" ]; then
+     if ! repquota -V &> /dev/null ; then
+        echo "no ('requota' executable is missing)"
+        exit 127
```

Comment:
munin expects the exitcode zero under all circumstances for `autoconf` (see the the [plugin documentation](http://munin-monitoring.org/wiki/PluginConcise#autoconf). The message (starting with "yes" or "no") is expected on stdout.